### ISSUE TITLE
Vaara: 008-kenttien virheen korjaus

### ISSUE THREAD

**Issue by pknuuti** [KohaSuomi/Koha#1316]
### Tukipyyntö

Vaarassa on n. 700 virheellistä tietuetta, joissa 008-kentässä 07-10 merkkipaikat ovat virheellisiä. Niitä ei voi korjata erämuokkauksella, koska kentässä on paljon muita merkkipaikkoja, joiden sisältö vaihtelee aineistoittain. Mutta tuon merkkipaikan virheet voisi korjata tietokannan kautta korvaamalla merkintä S.A. putkimerkeillä |||| ja vuosiluvuissa esiintyvä kysymysmerkki ? tai viiva - kirjaimella u (kuten formaatti määrää).
Esimerkit virheistä:
(008-kaikki/07-10 illegal value "S.A.", should be ["0-9"," ","u","||||"])
(008-kaikki/07-10 illegal value "197?", should be ["0-9"," ","u","||||"]) 
(008-kaikki/07-10 illegal value "19-?", should be ["0-9"," ","u","||||"])

**Comment by paxed** [comment]
Tämän saman ajon voinee tehdä kaikille kimpoille; outissa n. 2300 kpl, vaarassa ja vaskissa on n. 700 kpl kummassakin, lopuissa vain muutamia.

**Comment by pknuuti** [comment]
Tuskin ketään haittaa, että päästään virheistä eroon.

**Comment by AnttHeik** [comment]
Joo, OUTIssa voi ainakin tehdä saman ajon. Noilla merkkipaikoilla saattaa esiintyä myös muitakin kirjainyhdistelmiä, esim. "I.A."

**Comment by annavii** [comment]
Vaskissa voi tehdä myös.

**Comment by Hanna-Lumme** [comment]
Lumpeissa voi tehdä myös.

**Comment by mauriaittaniemi** [comment]
Lappiin voi tehdä myös.

**Comment by Kati-Helle** [comment]
Helleen myös.

**Comment by AnneliO** [comment]
Seuranta kimpoista:

- [ ] Lappi
- [ ] OUTI
- [ ] Vaara
- [ ] Lumme
- [ ] Helle
- [ ] Vaski

**Comment by pknuuti** [comment]
Vaarassa nämä virheet on korjattu. En tiedä, onko muissa kimpoissa. Joko tämän voi sulkea?

**Comment by AnneliO** [comment]
Merkitsin Vaaran valmiiksi, en tiedä muiden tilannetta.

**Comment by paxed** [comment]
Muutkin merkityt kimpat siivottu.

**Comment by pknuuti** [comment]
Tämä siis kunnossa, joten suljen tiketin.
